docs/UserGuide.md

```diff
@@ -64,129 +67,57 @@ AddressBook Level 3 (AB3) is a **desktop app for managing contacts, optimized fo
 
 </div>
 
-### Viewing help : `help`
-
-Shows a message explaning how to access the help page.
-
-![help message](images/helpMessage.png)
-
-Format: `help`
-
+### Adding a friend: `addfriend`
 
-### Adding a person: `add`
+Adds a new friend to the address book. A friend has:
+* ***(compulsory)*** name
+* *(optional)* phone number
+* *(optional)* email
+* *(optional)* address
+* *(optional)* description
 
-Adds a person to the address book.
+Format: `addfriend n/{NAME}  p/{PHONE_NUMBER} e/{EMAIL}  a/{ADDRESS} d/{DESCRIPTION}`
```

Comment:
We should standardize our use of {} and [] for optional parameters. Since UG already uses square brackets [] for the optional parameters I think we should use [] as well! Do check your other commands as well!

Reply:
Agreed! Updated UG to reflect the suggested changes